catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -44,46 +46,37 @@ const ClearSearch = styled.button`
 `;
 
 type SearchTagProps = {
+  link: NextLinkType,
   label: string,
-  name: string,
-  value: string,
   checked: boolean,
-  onChange: (event: SyntheticEvent<HTMLInputElement>) => void,
 };
-const SearchTag = ({
-  label,
-  name,
-  value,
-  checked,
-  onChange,
-}: SearchTagProps) => {
+const SearchTag = ({ link, label, checked }: SearchTagProps) => {
   return (
-    <label
-      className={classNames({
-        'bg-pumice': true,
-        'flex-inline': true,
-        'flex--v-center': true,
-        pointer: true,
-        [spacing(
-          { s: 1 },
-          { padding: ['left', 'right'], margin: ['left'] }
-        )]: true,
-        [font({ s: 'HNL3' })]: true,
-      })}
-      style={{ borderRadius: '3px' }}
-    >
-      <input
+    <NextLink {...link}>
+      <a
         className={classNames({
-          [spacing({ s: 1 }, { margin: ['right'] })]: true,
+          // 'bg-pumice': true,
+          'flex-inline': true,
+          'flex--v-center': true,
+          pointer: true,
+          [spacing(
+            { s: 1 },
+            { padding: ['left', 'right'], margin: ['left'] }
+          )]: true,
+          [font({ s: 'HNL4' })]: true,
         })}
-        type="checkbox"
-        name={name}
-        value={value}
-        checked={checked}
-        onChange={onChange}
-      />
-      {label}
-    </label>
+        style={{ borderRadius: '3px', textDecoration: 'underline' }}
+      >
+        <input
```

Comment:
I remembered earlier that the (badly named) `HTMLInput` component we created handles styled checkboxes and radios (e.g. the checkboxes are used on the [newsletter](https://wellcomecollection.org/newsletter) page).

Might be worth splitting out these into `Checkbox` and `Radio` components if we want to keep a bit of branding for these elements?

Reply:
I'll take a look at this next.